data/lenet_memory_solver.prototxt

```diff
@@ -1,9 +1,9 @@
 # The train/test net protocol buffer definition
 net: "lenet_memory_train_test.prototxt"
 # test_iter specifies how many forward passes the test should carry out.
-test_iter: 1
+test_iter: 2
```

Comment:
This pr is not ready for review yet

On Saturday, August 20, 2016, anfeng <notifications@github.com> wrote:

> In data/lenet_memory_solver.prototxt
> <https://github.com/yahoo/CaffeOnSpark/pull/135#discussion_r75587744>:
>
> > @@ -1,9 +1,9 @@
> >  # The train/test net protocol buffer definition
> >  net: "lenet_memory_train_test.prototxt"
> >  # test_iter specifies how many forward passes the test should carry out.
> > -test_iter: 1
> > +test_iter: 2
>
> We should not use this solver file in data folder. Unit tests etc should
> use the solver file in test/resources folder.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/yahoo/CaffeOnSpark/pull/135/files/cb9786af4569a868ca2a6817297e45d2b2df999c#r75587744>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ACCTVTj-TJXzFg_mphXCpmLZq1lyz7UOks5qh67sgaJpZM4JpLru>
> .
>